src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.ts

```diff
@@ -161,7 +119,12 @@ export class InstructorSessionsPageComponent extends InstructorSessionModalPageC
               private route: ActivatedRoute) {
     super(instructorService, statusMessageService, navigationService, feedbackSessionsService,
         feedbackQuestionsService, tableComparatorService, ngbModalService,
-        simpleModalService, progressBarService, feedbackSessionActionsService, timezoneService, studentService);
+      simpleModalService, progressBarService, feedbackSessionActionsService, timezoneService, studentService);
+
+    this.sessionEditFormModel = {
+      ...this.sessionEditFormModel,
+      isEditable: true,
```

Comment:
Shall we set the `gracePeriod` to 15 as well? Seems like that was lost upon moving the `sessionEditFormModel` to the base page component